lib/hyrax/configuration.rb

```diff
@@ -381,14 +381,20 @@ def google_analytics_id
 
     attr_writer :permission_levels
     def permission_levels
-      @permission_levels ||= { "Choose Access" => "none",
-                               "View/Download" => "read",
-                               "Edit" => "edit" }
+      @permission_levels ||= { "View/Download" => "read",
```

Comment:
Yeah the i18n work was a much larger change... that's actually why I brought up the question about it in standup before I went on vacation. I can do that if it's what you want, but was just trying to get this change through quickly.